src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -699,48 +690,6 @@ public void ValidateGraphQLURI(RuntimeConfig runtimeConfig)
         }
     }
 
-    /// <summary>
```

Comment:
Can you add in the PR description why was this change necessary?

Reply:
Added